libs/checkpoint-sqlite/langgraph/checkpoint/sqlite/aio.py

```diff
@@ -329,14 +330,14 @@ async def alist(
             AsyncIterator[CheckpointTuple]: An asynchronous iterator of matching checkpoint tuples.
         """
         await self.setup()
-        where, param_values = search_where(config, filter, before)
+        where, params = search_where(config, filter, before)
         query = f"""SELECT thread_id, checkpoint_ns, checkpoint_id, parent_checkpoint_id, type, checkpoint, metadata
         FROM checkpoints
         {where}
         ORDER BY checkpoint_id DESC"""
         if limit:
             query += f" LIMIT {limit}"
-        async with self.conn.execute(query, param_values) as cursor:
+        async with self.conn.execute(query, params) as cur, self.conn.cursor() as wcur:
```

Comment:
i think this needs to be `aclosing(self.conn.cursor()) as wcur`

Reply:
No, aiosqliue cursor actually is a context manager

Reply:
ah cool